pkgs/tools/misc/shelldap/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchurl, perlPackages }:
+perlPackages.buildPerlPackage rec {
+  name = "shelldap-${version}";
+  version = "1.4.0";
+  src = fetchurl {
```

Comment:
We have ```fetchFromBitBucket```

Reply:
It is not the same.
https://bitbucket.org/mahlon/shelldap/downloads are explicitly uploaded tarballs, not repo snapshots.

Reply:
So I guess I shouldn't change it right?